superdesk/users/users.py

```diff
@@ -15,112 +15,113 @@
 
 
 class UsersResource(Resource):
+    readonly = True
```

Comment:
This will make first_name,  email, last_name, etc as read_only for db authentication mode as well.